OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -158,6 +158,9 @@ public bool FaceTarget(Actor self, Target target)
 
 		public void StopAiming(Actor self)
 		{
+			if (attack == null)
```

Comment:
IMO it would be better to combine this into the check below: `if (attack != null && attack.IsAiming)`

Reply:
sure